new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1155 @@
+#             Welcome   to   the   VIM   Tutor
+
+Vim is a very powerful text editor that has many commands. Unlike most text
+editors where you use the keyboard to enter text and arrow keys to move your
+cursor, in Vim you use the keyboard to enter commands to manipulate text.
+These commands allow you to express complex thoughts in just a few keystrokes
+to move through text and manipulate it much faster than in most text editors.
+There is a great amount of commands, too many to explain in a tutor such as
+this, but as you will see over the course of this tutorial series these
+commands are not arbitrarily chosen, they form a sort of language with its own
+grammar.
```

Comment:
Yes, less is more in a tutorial, especially if it is the very first thing the reader is going to read. The "too many to explain" part was carried over from the previous version, I was being more conservative with my changes.